ubiconfig/_impl/loaders/gitlab.py

```diff
@@ -10,28 +10,30 @@
 
 from .base import Loader
 
-LOG = logging.getLogger('ubiconfig')
+LOG = logging.getLogger("ubiconfig")
 
 
 class GitlabLoader(Loader):
     """Load configuration from a remote repo on gitlab."""
+
     def __init__(self, url):
         """
         :param url: gitlab repo url in form of `https://<host>/<repo>`
         """
         self._url = url
         self._session = requests.Session()
-        self._repo_api = RepoApi(self._url.rstrip('/'))
+        self._repo_api = RepoApi(self._url.rstrip("/"))
         self._files_branch_map = self._pre_load()
 
     def load(self, file_name):
         """ Load file from remote repository.
         :param file_name: filename that is on remote repository in any branch
         """
         # find the right branch from the mapping
-        branch = self._files_branch_map[file_name]
+        branch_sha1 = self._files_branch_map[file_name]
+        version = branch_sha1[0][3:]  # such as ubi7, ubi7.1
```

Comment:
I wouldn't rely on current branch names, it can easily change in future, so I guess using regexp would be better.

Reply:
I don't get the point it can be easily changed in the future, do you mean next milestone? I expect it won't be changed easily.

Reply:
Well, I meant that name of branch is not defined so it doesn't have to contain just three letters.
But anyway in near future we are going to work only with "ubiX.Y" names, so I guess it's ok for now.